src/main/java/org/embulk/output/sftp/SftpUtils.java

```diff
@@ -310,6 +314,15 @@ FileObject newSftpFile(final URI sftpUri) throws FileSystemException
         if (file.exists()) {
             file.delete();
         }
+
+        logger.info("Current file path: {}", file.getPublicURIString());
+
+        FileObject parent = file.getParent();
```

Comment:
- Parallel execution does not generate files in the specified path.
  - `Current file path: sftp://sftp-user@18.182.4.48/`

```
2023-09-13 08:38:49.389 +0000 [INFO] (0036:task-0000): Current file path: sftp://sftp-user@18.182.4.48/test_by_koshikawa/00.00.json
2023-09-13 08:38:49.392 +0000 [INFO] (0036:task-0000): parent directory sftp://sftp-user@18.182.4.48/test_by_koshikawa exists there
2023-09-13 08:38:49.395 +0000 [INFO] (0036:task-0000): Uploading to remote sftp file (7 KB): sftp://sftp-user@18.182.4.48/test_by_koshikawa/00.00.json
2023-09-13 08:38:49.417 +0000 [INFO] (0036:task-0000): Upload completed.
2023-09-13 08:38:49.773 +0000 [INFO] (0036:task-0000): Current file path: sftp://sftp-user@18.182.4.48/
2023-09-13 08:38:49.774 +0000 [WARN] (0036:task-0000): Unable to retrieve the parent directory for sftp://sftp-user@18.182.4.48:22. Skipping further processing.
```